app.js

```diff
@@ -0,0 +1,99 @@
+const School = require("./School.js")
+const express = require("express")
+const cors = require("cors")
+const app = express()
+const port = 7000
+//const usersRouter = require("./School.js")
+const bodyParser = require("body-parser")
+
+app.use(bodyParser.urlencoded({extended:false}))
+app.use(bodyParser.json())
+app.use(cors())
+
+//app.use("/school", usersRouter)
+//app.use("/class", usersRouter)
+
+let mySchool = new School()
+
+// App.post for creating a new class
+// const checkIfClassExists = (req, res, next) =>{
+//     if (req.bl)
+// }
+
+
+
+app.get('/class', (req, res)=>{
+    let classes = mySchool.classes
+    res.json({
+        classes
+    })
+})
+
+
+app.get('/class/student', (req, res)=>{
+    let students = Student.name
```

Comment:
Student.name would throw an error, you want to get the name from the req.params, or req.query